src/main/java/org/broadinstitute/consent/http/util/CountryValidator.java

```diff
@@ -10,10 +10,27 @@
 import java.nio.charset.StandardCharsets;
 import java.util.HashSet;
 import java.util.Set;
+import org.broadinstitute.consent.http.models.DataAccessRequest;
 
 public class CountryValidator {
   private static final String FILEPATH = "assets/ISO-3166-countries.json";
   private Set<String> countries;
+
+  // For the list of countries banned, see
+  // https://www.ecfr.gov/current/title-28/chapter-I/part-202/subpart-F/section-202.601
+  private static final Set<String> bannedCountriesISO3166 =
+      Set.of(
+          "china",
+          "cuba",
+          "hong kong",
+          "iran, (islamic republic of)",
+          "Korea (the democratic people's republic of)",
+          "macao",
+          "russian federation (the)",
+          "venezuela, (bolivarian republic of)");
+  private static final Set<String> bannedCountriesCFR =
+      Set.of("china", "cuba", "hong kong", "iran", "Macao", "north korea", "russia", "venezuela");
```

Comment:
ISO-3166 has only as "Macao", but [28 CFR Part 202](https://www.ecfr.gov/current/title-28/chapter-I/part-202) has both "Macau" (4 times) and "Macao" (1 time).

To account for both spelling variants:

```suggestion
      Set.of("china", "cuba", "hong kong", "iran", "Macao", "Macau", "north korea", "russia", "venezuela");
```

Having both here seems ideal, but in user-facing prose, "Macau" seems closer to the CFR.

Reply:
We have a ground truth list in `src/main/resources/assets/ISO-3166-countries.json` for both our form elements and input validation in the backend.  We only accept these values.  See validateCountryOfOperation in DataAccessRequestService.

Reply:
Added entry for Macau to the CFR restriction list.